woocommerce/wc-admin
# Add nvmrc for newest lts release

This adds an .nvmrc file which points to the latest lts release of node.
For those people who use nvm, this will keep them current on the latest
stable version.

## To test:
1. Ensure `nvm` is installed.
2. `cd` into wc-admin directory.
3. nvm should switch to latest LTS version (v10.14.2 as of Dec. 17th)
